Sources/NIOSSH/Child Channels/ChildChannelOptions.swift

```diff
@@ -53,4 +53,11 @@ extension SSHChildChannelOptions.Types {
 
         public init() {}
     }
+    
+    /// `UsernameOption` allows users to query the authenticated username of the channel.
+    public struct UsernameOption: ChannelOption {
+        public typealias Value = String?
+        
+        public init() {}
+    }
```

Comment:
Can you add the associated `static let` above?

Reply:
Done